lib/src/revset.rs

```diff
@@ -2282,6 +2346,13 @@ mod tests {
             parse("foo.bar-v1+7-"),
             Ok(RevsetExpression::symbol("foo.bar-v1+7".to_string()).parents())
         );
+        // Default arguments for *branches() are all ""
+        assert_eq!(parse("branches()"), parse("branches(\"\")"));
+        assert_eq!(parse("remote_branches()"), parse("remote_branches(\"\")"));
+        assert_eq!(
+            parse("remote_branches()"),
+            parse("remote_branches(\"\", \"\")")
+        );
```

Comment:
Nit: you can use `r#""` to avoid backslashes.

Reply:
Again, TIL!